packages/hdwallet-provider/test/provider.test.ts

```diff
@@ -65,7 +65,7 @@ describe("HD Wallet Provider", function () {
           "http://localhost:8545"
         );
         assert.fail("Should throw on invalid mnemonic");
-      } catch (e) {
+      } catch (e: any) {
```

Comment:
Do these even need to be typed? They all should be of type `Error`, no?

Reply:
IIRC, typed catch handlers are now required when `noImplicitAny` is `true` (as it is in all of our `tsconfig.json`) files.

Reply:
Also FWIW, I've verified that if any code changes here were reverted, the build would break.

Reply:
Ok that makes sense. But even so, they should be of type `Error` right?

Reply:
No, there is no requirement to throw an object of type `Error` in JavaScript. You can throw whatever you like. This is why the TS compiler only allows a type of `any` here.

Reply:
I stand corrected, it can also be type `unknown`.

Reply:
Apparently [best practice](https://typescript.tv/best-practices/error-ts1196-catch-clause-variable-type-annotation/) is to use `unknown` with type guards when you find yourself wishing that you had Java or C# like typed error handlers, but given that this is existing code that isn't throwing due to catching a type with a missing `message` property, I think that'd be complete overkill here.